src/librustdoc/html/static/playpen.js

```diff
@@ -15,6 +15,11 @@ document.addEventListener('DOMContentLoaded', function() {
     'use strict';
 
     if (!window.playgroundUrl) {
+        var run_buttons = document.querySelectorAll(".test-arrow");
```

Comment:
I used trollface to make sure people understand it was a troll. I failed :-/

I'll update later on.